pkg/mock/mockoption/mockoption.go

```diff
@@ -183,6 +186,11 @@ func (mso *ScheduleOptions) GetStrictlyMatchLabel() bool {
 	return mso.StrictlyMatchLabel
 }
 
+// GetEnablePlacementRules mocks method
+func (mso *ScheduleOptions) GetEnablePlacementRules() bool {
```

Comment:
would you like replace it with `isPlacementRulesEnabled`？

Reply:
Good point